content/src/arrows.js

```diff
@@ -0,0 +1,52 @@
+///////////////////////////////////////////////////////////////////////////
+// Arrow Support to Create Code-flow Arrows.
```

Comment:
So, am I correct in understanding that this particular file is just for *drawing* an arrow given where to draw it; not about deciding where and when to place the arrows?

Could you make the comment at the top a bit more specific so it describes what exactly this does or does not do with arrows? and maybe add a comment somewhere describing where this actually gets used?
Otherwise it's a bit hard for me to browse through the code and figure out how it's all connected.